region/norfair/west/Crocomire Escape.json

```diff
@@ -254,6 +254,192 @@
+          {"ammo": {"type": "Super", "count": 1}},
+          {"obstaclesCleared": ["A"]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithGModeSetup": {}
+      },
+      "collectsItems": [3],
+      "unlocksDoors": [
+        {
+          "types": ["ammo"],
+          "requires": []
+        }
+      ],
+      "flashSuitChecked": true,
+      "note": "Lure the Geruta to the left door. This may require many jumps to get it to swoop up onto the ledge and not get stuck.",
+      "devNote": "It's possible to avoid the item, but the Geruta will get stuck on it and usually require many swoops to get past it."
+    },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Setup - Get Hit By Geruta, Heatproof, Morphless",
+      "requires": [
+        "h_heatProof",
+        "canBePatient",
+        "canCameraManip",
+        {"or": [
+          "canInsaneJump",
+          "Ice",
+          "h_lavaProof",
+          {"enemyDamage": {
+            "enemy": "Geruta",
+            "type": "contact",
+            "hits": 1
+          }},
+          {"and": [
+            "canNeutralDamageBoost",
+            {"enemyDamage": {
+              "enemy": "Dragon",
+              "type": "fireball",
+              "hits": 1
+            }},
+            {"lavaFrames": 55}
+          ]}
+        ]},
+        {"or": [
+          "canTrickyDodgeEnemies",
+          "SpaceJump",
+          "Charge",
+          "Ice",
+          "Plasma",
+          {"ammo": {"type": "Missile", "count": 2}},
+          {"ammo": {"type": "Super", "count": 2}},
+          {"enemyDamage": {
+            "enemy": "Dragon",
+            "type": "fireball",
+            "hits": 5
+          }},
+          {"and": [
+            "h_lavaProof",
+            "ScrewAttack"
+          ]}
+        ]},
+        {"or": [
+          "SpaceJump",
+          {"and": [
+            "HiJump",
+            "SpeedBooster",
+            "canWalljump"
+          ]},
+          {"and": [
+            "canBeVeryPatient",
+            "HiJump",
+            "Grapple"
+          ]}
+        ]},
+        {"or": [
+          {"ammo": {"type": "Super", "count": 1}},
+          {"obstaclesCleared": ["A"]}
+        ]}
+      ],
+      "exitCondition": {
+        "leaveWithGModeSetup": {}
+      },
+      "collectsItems": [3],
+      "unlocksDoors": [
+        {
+          "types": ["ammo"],
+          "requires": []
+        }
+      ],
+      "flashSuitChecked": true,
+      "note": [
+        "Lure the Geruta to the left door. This may require many jumps to get it to swoop up onto the ledge and not get stuck.",
+        "To safely pass the Geruta without a hit, jump left below it, to get it to swoop left before going up."
+      ],
+      "devNote": "It's possible to avoid the item, but the Geruta will get stuck on it and usually require many swoops to get past it."
+    },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Setup - Get Hit By Geruta, Heatproof, Frozen Geruta Platform",
```

Comment:
The name and description can be confused with riding the geruta across while freezing it repeatedly.

Reply:
I forgot about that one, but I'm not including it, because it requires many more swoops and hits than just crossing the room, as it can take a while to get it in the top left region